server/appconfig.ts

```diff
@@ -1,18 +1,18 @@
-const express = require("express"),
-bodyParser = require("body-parser"),
+import { Request, Response, NextFunction } from 'express';
+const bodyParser = require("body-parser"),
 errorMessages = require("./api/utils/errorMessages");
 
-module.exports.initialize = function(app) {
+module.exports.initialize = function(app: any) {
 
-    function appUse(app, ...arg) {
+    function appUse(app: any, ...arg: any[]) {
```

Comment:
We shouldn't use `any` type here! Let's import `Application` from express and cast the `app: Application`!